### DISCUSSION TITLE
Simple Login Thunderbird Add-on

### DISCUSSION THREAD

**Discussion by GammelSami** [simple-login/app#621] [Feature Requests]
When klicking on a mailto link, I often forget to create a reverse alias. 
That's confusing for the recipient and bad for my privacy.

Would be cool to have a Thunderbird Add-on that (at least) asks me if I really wanna not use a reverse alias before sending.
More advanced features could follow in the future.

**Comment by WhamboMPS** [comment]
@GammelSami, FWIW I totally get the benefit of this feature request. The problem is there are _a lot_ more email reader environments than there are web browser / mobile app environments. For instance, see this partial list of [email clients](https://en.wikipedia.org/wiki/Comparison_of_email_clients) on Wikipedia (and even as broad as it is this list doesn't include Gmail which has fat clients for mobile devices!)

Email client specific add-ons would be a great thing to have. It may also be a [Sisyphean](https://en.wikipedia.org/wiki/Sisyphus) task to accomplish.

**Comment by software-medic** [comment]
Can you create a SimpleLogin Mozilla Thunderbird Addon?

I find the back and forth of using a web browser and the Mozilla Thunderbird application for creating reverse aliases a bit tedious.
Perhaps integrate a way to have a 'Contacts List interface in the Addon for creating and managing reverse aliases.

Use Case:
Overtime most people have managed to accumulated several email accounts from different email providers.
Using Mozilla Thunderbird client as a unified client for all these inboxes with a single click to open and master password is very useful.

**Reply by GammelSami** [reply]
Well, as first step, it would be very easy to just have an addon that opens app.simplelogin.io in a second Thunderbird tab, i think.

**Comment by Edontin** [comment]
Modern Thunderbird versions [support similar WebExtension APIs to Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Developing_WebExtensions_for_Thunderbird). This seems like it might make the task a bit more feasible, at least superficially.
For example, uBlock Origin supports both Firefox and Thunderbird (though there are differences in `manifest.json`), and can handle ad blocking within the integrated Thunderbird web browser.

**Comment by benjuade** [comment]
I found two useful addons for the proton ecosystem:

- https://github.com/oktupol/additional-info-for-simplelogin-thunderbird
- (if you use Proton) https://github.com/Dan-Q/ProtonMail-Encryption-Indicators-for-Thunderbird
